### PR TITLE
Initial Release

### PR DESCRIPTION
**Pull request by elian-kurtenkov**

### CODE REVIEW

File: client/Lykke.Service.Pledges.Client/AutorestClientMapper.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Service.Pledges.Client
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.Pledges.Client/AutorestClientMapper.cs]: Delete an empty class if it is not used.

File: client/Lykke.Service.Pledges.Client/IPledgesClient.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿
+using Lykke.Service.Pledges.Client.AutorestClient.Models;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.Pledges.Client
+{
+    public interface IPledgesClient
+    {
+        Task Create(CreatePledgeRequest request);
```
Comment (MikhailGerasimov) [inline client/Lykke.Service.Pledges.Client/IPledgesClient.cs]: Please add a description.

File: client/Lykke.Service.Pledges.Client/IPledge.cs [hunk 3]
```diff
@@ -0,0 +1,6 @@
+﻿namespace Lykke.Service.Pledges.Client
+{
+    public interface IPledge
```
Comment (samodovdi) [inline client/Lykke.Service.Pledges.Client/IPledge.cs]: can be removed

File: src/Lykke.Service.Pledges.Core/Settings/ServiceSettings/ServiceSettings.cs [hunk 4]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Lykke.Service.Pledges.Core.Settings.ServiceSettings
+{
+    public class ServiceSettings
+    {
+        public string ClientAccountServiceUrl { get; set; }
```
Comment (samodovdi) [inline src/Lykke.Service.Pledges.Core/Settings/ServiceSettings/ServiceSettings.cs]: use it as a separate class like :
public class ClientAccountService
{
   public string ServiceUrl {get;set;}
}
and use it as a property in AppSettings:
public ClientAccountService ClientAccountClient {get;set;}

so you'll end up with the json settings like this (a top level property for the client account service) - http://13.95.227.175/e7b6cb180ad24cb3bf4c84974f96872a_LykkeRegistration
and will use access token ClientAccountClient.* on the service settings

**Review by samodovdi — APPROVED** [review]

File: client/Lykke.Service.Pledges.Client/IPledge.cs (hunk 3 above)
Comment (filip-prime) [reply client/Lykke.Service.Pledges.Client/IPledge.cs]: Done

File: src/Lykke.Service.Pledges.Core/Settings/ServiceSettings/ServiceSettings.cs (hunk 4 above)
Comment (filip-prime) [reply src/Lykke.Service.Pledges.Core/Settings/ServiceSettings/ServiceSettings.cs]: Done

**Pull request merged**